api/features/permissions.py

```diff
@@ -66,6 +72,31 @@ def has_object_permission(self, request, view, obj):
         )
 
 
+class MasterAPIKeyFeatureStatePermissions(BaseHasAPIKey):
```

Comment:
Doesn't seem like we are using anything from the `BaseHasAPIKey` class? We're overriding the 2 permission methods and not using the `get_key` method.